asdf/yamlutil.py

```diff
@@ -318,11 +324,11 @@ def dump_tree(tree, fd, ctx):
 
     ctx : Context
         The writing context.
-    """
-    class AsdfDumperTmp(AsdfDumper):
```

Comment:
This is just a thing I noticed while I was here.  At some point I changed `AsdfDumper` to not use the `ctx` variable, but then forgot to remove this special class definition.